src/style.css

```diff
@@ -1 +1,47 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src:
+    url(fonts/Avenir-Book.ttf) format(ttf),
+    url(fonts/Avenir-Heavy.ttf) format(ttf),
+    url(fonts/Avenir.ttc) format(ttc);
+}
+
+.search-bar {
+  display: flex;
+  width: 100%;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px #3d4e611a;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  border: solid 1px #e1e7ed;
+  border-radius: 3px;
+  font-family: Avenir, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. It's a good practice to specify a generic family as a fallback, such as 'sans-serif' for Avenir in case it fails to load.